tests/test_serialization.py

```diff
@@ -136,6 +148,9 @@ def test_pickle_list() -> None:
         fp.seek(0)
         c1 = pickle.load(fp)
         assert c == c1
+        assert get_ref_type(c1) == Optional[List[Any]]
+        assert c1._metadata.element_type is Any
+        assert c1._metadata.optional is True
```

Comment:
You are testing this in the following test, right?
should those two tests be merged?